extensions/wikia/Chat2/resourceloader/ResourceLoaderChatUserModule.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Resource Loader module for user JS and CSS for inclusion in Special:Chat
+ */
+class ResourceLoaderChatUserModule extends ResourceLoaderUserModule {
+
+	protected function getPages( ResourceLoaderContext $context ) {
+		$userName = $context->getUser();
+
+		if ( $userName ) {
+			$userPageTitle = Title::makeTitleSafe( NS_USER, $userName );
+			$userPageText = $userPageTitle->getPrefixedDBkey();
+
+			$pages = [
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$pages". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)